app/helpers/application_helper/toolbar/drift_view.rb

```diff
@@ -11,7 +11,8 @@ class ApplicationHelper::Toolbar::DriftView < ApplicationHelper::Toolbar::Basic
       'fa fa-bars fa-rotate-90 fa-lg',
       N_('Compressed View'),
       nil,
-      :url => "drift_compress"),
+      :url   => "drift_compress",
```

Comment:
I'd leave it as it is now, just add also the class to `drift_expanded` button, there's issue right now, the button is hidden.